test/library/standard/GMP/ldelaney/gmp_Bigint_operators.good

```diff
@@ -80,7 +80,7 @@
  12 -    25 =         -13
 
  25 -   -13 =          38
--13 -    25 =          38
```

Comment:
What the - how did this one slip in in the first place? xD

Reply:
The other PRs I've merged have had issues in this test, it seems to be a case of a write the output to the `.good` file that I'm sure we've all been guilty of... Thanks for the review!